.github/workflows/bump-version.yaml

```diff
@@ -3,6 +3,21 @@ on:
   push:
     branches:
       - main
+  workflow_call:
+    secrets:
+      SLACK_BOT_TOKEN:
+        description: >
+          The Slack API bot token for your custom app to issue a notification
+          from with the result of the workflow. The token must have the
```

Comment:
I think I understand what you're saying here but the "from with" feels awkward and hard to understand.  There is probably a way to rephrase this that would be clearer.

Reply:
Good point; take two!